app/src/main/java/com/sschuraytz/charcoaldrawing/EraseTool.java

```diff
@@ -1,10 +1,12 @@
 package com.sschuraytz.charcoaldrawing;
 
 
+import android.graphics.Color;
+
 public class EraseTool extends Tool{
 
     public EraseTool() {
-        super(0xFAFAFA);
+        super(Color.WHITE);
```

Comment:
Why is this white now?

Reply:
> Why is this white now?

It was originally white, but then I noticed that because the default background is not pure white, in certain lighting, erase was adding a new color to the canvas instead of recreating the background color. So, I figured out a color that would blend in with the background. But now that I'm creating a background bitmap to avoid a black background when saving, I control the color. So, it can be whatever I choose as long as the background and eraser match. I thought white made sense, but if you think it's nice to have the more natural feel of off-white, I'm open to switching.

(You have to hold the phone at the right angle to see the subtle color difference. It didn't come through in the original screenshot. So, I'm also including a version with manipulated lighting to make it more clear.)
![eraser_vs_background_white](https://user-images.githubusercontent.com/44480876/94220113-21496480-fe9d-11ea-9d36-afb8b87a16ab.png)
![eraser_vs_background_white_adjustedLighting](https://user-images.githubusercontent.com/44480876/94220119-23abbe80-fe9d-11ea-90b3-abc698a339c0.png)